packages/babel-node/test/index.js

```diff
@@ -1,200 +1,22 @@
-    } else {
-      expect(stderr).toBe(expectStderr);
-    }
-  } else if (stderr) {
-    throw new Error("stderr:\n" + stderr + "\n\nstdout:\n" + stdout);
-  }
-
-  const expectStdout = opts.stdout.trim();
-  stdout = stdout.trim();
-  stdout = stdout.replace(/\\/g, "/");
-
-  if (opts.stdout) {
-    if (opts.stdoutContains) {
-      expect(stdout.includes(expectStdout)).toBeTruthy();
-    } else {
-      expect(stdout).toBe(expectStdout);
-    }
-  } else if (stdout) {
-    throw new Error("stdout:\n" + stdout);
-  }
-
-  if (opts.outFiles) {
-    const actualFiles = readDir(path.join(tmpLoc));
-
-    Object.keys(actualFiles).forEach(function (filename) {
-      if (!Object.prototype.hasOwnProperty.call(opts.inFiles, filename)) {
-        const expected = opts.outFiles[filename];
-        const actual = actualFiles[filename];
-
-        expect(expected).not.toBeUndefined();
-
-        if (expected) {
-          expect(actual).toBe(expected);
-        }
-      }
-    });
-
-    Object.keys(opts.outFiles).forEach(function (filename) {
-      expect(actualFiles).toHaveProperty(filename);
-    });
-  }
-};
-
-const buildTest = function (binName, testName, opts) {
-  const binLoc = path.join(dirname, "../lib", binName);
-
-  return function (callback) {
-    saveInFiles(opts.inFiles);
-    let args = [binLoc];
-    args.push("--config-file", "../config.json");
-    args = args.concat(opts.args);
-
-    const spawn = child.spawn(process.execPath, args);
-
-    let stderr = "";
-    let stdout = "";
+const binLoc = path.join(dirname, "../bin/babel-node.js");
+const childLoc = path.join(dirname, "fixtures/misc/child.js");
 
-    spawn.stderr.on("data", function (chunk) {
-      stderr += chunk;
-    });
-
-    spawn.stdout.on("data", function (chunk) {
-      stdout += chunk;
-    });
-
-    spawn.on("close", function () {
-      let err;
-
-      try {
-        assertTest(stdout, stderr, opts);
-      } catch (e) {
-        err = e;
-      }
-
-      if (err) {
-        err.message =
-          args.map(arg => `"${arg}"`).join(" ") + ": " + err.message;
-      }
-
-      callback(err);
-    });
-
-    if (opts.stdin) {
-      spawn.stdin.write(opts.stdin);
-      spawn.stdin.end();
-    }
-  };
-};
-
-fs.readdirSync(fixtureLoc).forEach(function (binName) {
-  if (binName[0] === ".") return;
-
-  const suiteLoc = path.join(fixtureLoc, binName);
-  describe("bin/" + binName, function () {
-    let cwd;
+describe("babel-node", () => {
+  it("ipc works with spawned babel-node process", done => {
+    expect.assertions(1);
 
-    beforeEach(() => {
-      cwd = process.cwd();
-
-      if (fs.existsSync(tmpLoc)) {
-        for (const child of fs.readdirSync(tmpLoc)) {
-          rimraf.sync(path.join(tmpLoc, child));
-        }
-      } else {
-        fs.mkdirSync(tmpLoc);
-      }
-
-      process.chdir(tmpLoc);
-    });
-
-    afterEach(() => {
-      process.chdir(cwd);
+    const child = spawn(binLoc, [childLoc], {
```

Comment:
The shebang in `bin/babel-node` may not work on Windows, consider invoke `babel-node` via `node`.